lib/server.js

```diff
@@ -20,34 +21,29 @@ const { clearRegularUpdateCache } = require('./regular-update')
 const optionalUrl = Joi.string().uri({ scheme: ['http', 'https'] })
 const requiredUrl = optionalUrl.required()
 
-const configSchema = Joi.object({
+const publicConfigSchema = Joi.object({
   bind: {
-    port: Joi.number()
-      .port()
-      .required(),
-    address: Joi.alternatives()
-      .try(
-        Joi.string()
-          .ip()
-          .required(),
-        Joi.string()
-          .hostname()
-          .required()
-      )
-      .required(),
+    port: Joi.number().port(),
+    address: Joi.alternatives().try(
+      Joi.string()
+        .ip()
+        .required(),
+      Joi.string()
+        .hostname()
+        .required()
+    ),
   },
   metrics: {
     prometheus: {
       enabled: Joi.boolean().required(),
-      allowedIps: Joi.string(),
+      allowedIps: Joi.array().required(),
```

Comment:
Is it worth adding the `Joi.string().ip()` validation on the individual array items?

Reply:
Done